packages/magento-product/components/ProductFiltersPro/ProductFilterRangeSection.tsx

```diff
@@ -25,38 +26,51 @@ export function ProductFilterRangeSection(props: FilterProps) {
   if (!options) return null
 
   return (
-    <Controller
-      control={control}
-      name={name}
-      render={({ field: { onChange, value } }) => {
-        if (value && !isFilterTypeRange(value)) throw new Error('Invalid filter type')
-
-        const from = value?.from ? Number(value?.from) : min
-        const to = value?.to ? Number(value?.to) : max
-
-        return (
-          <Box sx={{ my: 2 }}>
-            <SectionHeader
-              labelLeft={label}
-              sx={{ mt: 0 }}
-              labelRight={
-                from !== min || to !== max ? (
-                  <Button
-                    variant='inline'
-                    color='primary'
-                    onClick={() => {
-                      form.resetField(name, { defaultValue: null })
-                    }}
-                  >
-                    <Trans id='Clear' />
-                  </Button>
-                ) : undefined
+    <Box
+      sx={(theme) => ({
+        [theme.breakpoints.down('md')]: {
+          display: 'none',
+        },
+      })}
+    >
+      <Controller
+        control={control}
+        name={name}
+        render={({ field: { onChange, value } }) => {
+          if (value && !isFilterTypeRange(value)) throw new Error('Invalid filter type')
+
+          const from = value?.from ? Number(value?.from) : min
+          const to = value?.to ? Number(value?.to) : max
+
+          return (
+            <ProductFilterAccordion
+              summary={
+                <Box sx={{ my: 2 }}>
+                  <SectionHeader
```

Comment:
Maybe we can drop the SectionHeader completely as we're using the accordion now? An accordion also allows additional items on the right.